lang/en-gb.xml

```diff
@@ -150,6 +150,14 @@ Do not power off the system or close the application.</warning_saving>
 		<trophy_earned>You have earned a trophy!</trophy_earned>
 	</indicator>
 
+	<initial_setup>
+		<back>Back</back>
+		<completed_setup>You have now completed initial setup.
+Your PlayStation®Vita system is ready!</completed_setup>
```

Comment:
Don't use the PlayStation Vita name here. The user isn't setting up a PS Vita but Vita3K.

Reply:
No is fine 👌, is no problème

Reply:
i have changed vita for Vita3K ^^